components/AdvancedOptionsItem.qml

```diff
@@ -14,14 +14,14 @@ RowLayout {
 
     RowLayout {
         id: titlecolumn
-        Layout.alignment: Qt.AlignTop | Qt.AlignLeft
-        Layout.preferredWidth: 195
-        Layout.maximumWidth: 195
+        Layout.alignment: Qt.AlignVCenter | Qt.AlignLeft
+        property var language: persistentSettings.language
+        Layout.preferredWidth: language == "Deutsch" ? 280 : language == "Español" ? 250 : language == "Français" ? 235 : 220
+        Layout.maximumWidth: language == "Deutsch" ? 280 : language == "Español" ? 250 : language == "Français" ? 235 : 220
```

Comment:
A nitpick, but since it's equivalent to line 19, how about:

`Layout.maximumWidth: Layout.preferredWidth` ?